content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst

```diff
@@ -46,40 +54,40 @@ Two-step flow
-  by different carriers or shipping docks before being shipped.
-- Minimum requirement to use lot numbers or serial numbers to track products with a
-  :abbr:`FIFO (First In, First Out)`, :abbr:`LIFO (Last In, First Out)` or
-  :abbr:`FEFO (First Expired, First Out)` removal strategy.
-- Recommended for larger warehouses with high stock levels, or when stocking large items (such as
-  mattresses, large furniture, heavy machinery, etc.).
+- **Pick + ship**: bring products to an output location before shipping. Packages can be organized
+  by different carriers or moved to shipping docks before being shipped.
+- Minimum requirement to use lot numbers or serial numbers to track products with a :abbr:`FIFO
+  (First In, First Out)`, :abbr:`LIFO (Last In, First Out)` or :abbr:`FEFO (First Expired, First
+  Out)` removal strategy.
 - Products received will not be available for manufacturing, shipping, etc., until they are
-  transferred into stock.
+  transferred into warehouse stock.
+- Recommended for larger warehouses with high stock levels, or when stocking large items (e.g.,
+  mattresses, large furniture, heavy machinery, etc.).
 
 .. seealso::
-   :ref:`Process receipts and deliveries in two steps <inventory/receipts_delivery_two_steps>`
+   :doc:`receipts_delivery_two_steps`
 
 Three-step flow
 ---------------
 
 The receiving and shipping rules for a three-step configuration are as follows:
 
-- **Input + quality + stock**: Receive products at the input location, transfer them to a quality
+- **Input + quality + stock**: receive products at the input location, transfer them to a quality
   control area, and move the ones that pass inspection into stock.
-- **Pick + pack + ship**: Pick products according to their removal strategy, pack them in a
+- **Pick + pack + ship**: pick products according to their removal strategy, pack them in a
   dedicated packing area, and bring them to an output location for shipping.
-- Can be used when tracking products by lot or serial numbers when using a :abbr:`FIFO (First In,
+- Can be used when tracking products by lot or serial numbers when using :abbr:`FIFO (First In,
   First Out)`, :abbr:`LIFO (Last In, First Out)`, or :abbr:`FEFO (First Expired, First Out)` removal
   strategy.
-- Recommended for very large warehouses with very high stock levels.
-- Required for any warehouse needing to perform quality control inspections before receiving items
-  into stock.
 - Products received will not be available for manufacturing, shipping, etc., until they are
   transferred into stock.
+- Required for any warehouse needing to perform quality control inspections before receiving items
+  into stock.
+- Recommended for very large warehouses with very high stock levels.
 
 .. seealso::
-   - :ref:`Process receipts in three steps <inventory/receipts_three_steps>`
-   - :ref:`Process deliveries in three steps <inventory/delivery_three_steps>`
+   - :doc:`receipts_three_steps`
+   - :doc:`delivery_three_steps`
```

Comment:
After reading through this doc, I noticed that it doesn't contain specific information, and is more of a comparison doc between the different receipt and delivery types. 

If it's not too much trouble, what do you think of moving the contents of this doc into `content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations.rst`, the parent folder of this doc, like your reservation methods doc! Let me know what you think! :D

Reply:
I think that's a good idea - I think I'm going to leave this PR as an [IMP] for content-specific updates, though, and once it's merged I'll push up a new restructure to tackle that in a separate PR. Thanks for the suggestion!